src/traits/impls.rs

```diff
@@ -138,24 +166,25 @@ mod test {
         let header: Vec<u8> = vec![0, 0, 3];
         let issuer = keypair.public.to_bytes().to_vec();
         let holder = vec![0x15; 32];
-
         let payload: Vec<u8> = [header, issuer, holder, test_domain_data()].concat();
-        let signature = keypair.sign(context.bytes(&payload));
-
-        let encoded: Vec<u8> = [payload, signature.to_bytes().to_vec()].concat();
+        let signature_bytes = keypair.sign(context.bytes(&payload)).to_bytes().to_vec();
```

Comment:
This test is currently signing using some helper functions.
Goal is to test whether the new signing functions are working:
```
// it's unsigned
let signature  = H512::default();
let encoded: Vec<u8> = [payload, signature.to_bytes().to_vec()].concat();

let mut doughnut: ParityDoughnutV0 =
            Decode::decode(&mut &encoded[..]).expect("It is a valid doughnut");

// sign the doughnut
let secret_key = keypair.secret.to_ed25519_bytes();
let _ = doughnut.sign_sr25519(&secret_key).expect("it signed ok");

// verify the doughnut signature
assert_eq!(doughnut.verify(), Ok(()));
```